{"name": "test_file_hg", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <Test_ziyun.test_DNC.Test_device_ml object at 0x00000199792ECE10>\n\n    @allure.story(\"文件回滚\")\n    def test_file_hg(self):\n        rp_resp = requests.put(url + '/rollback',\n                               json={'fileId': fileId, 'versionId': versionId, 'reason': '自动化回滚'},\n                               headers={'token': token_data})\n        assertions.assert_code(rp_resp.status_code, 200)\n        rp_resp_json = rp_resp.json()\n        print(rp_resp_json)\n>       assertions.assert_in_text(rp_resp_json['message'], 'Success')\n\ntest_DNC.py:181: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Common.Assert.Assertions object at 0x00000199782894E0>\nbody = '未查找到该文件信息', expected_msg = 'Success'\n\n    def assert_in_text(self, body, expected_msg):\n        \"\"\"\n        验证response body中是否包含预期字符串\n        :param body:\n        :param expected_msg:\n        :return:\n        \"\"\"\n        try:\n            text = json.dumps(body, ensure_ascii=False)\n            # print(text)\n>           assert expected_msg in text\nE           AssertionError\n\n..\\Common\\Assert.py:59: AssertionError"}, "attachments": [{"name": "log", "source": "d3be3c17-3bbe-4d5d-a6b6-e3e90db8d9f3-attachment.txt", "type": "text/plain"}], "start": 1600411754975, "stop": 1600411755389, "uuid": "ea78a485-dd20-435c-a2bc-61c258e7806b", "historyId": "93f35517ee710fa17a40eadd2c2a4ecf", "testCaseId": "9f605f3bce2323be60cce5cd9b121dcb", "fullName": "test_DNC.Test_device_ml#test_file_hg", "labels": [{"name": "story", "value": "文件回滚"}, {"name": "feature", "value": "设备目录分发页、回传页接口"}, {"name": "suite", "value": "test_DNC"}, {"name": "subSuite", "value": "Test_device_ml"}, {"name": "host", "value": "DESKTOP-FIOLAUQ"}, {"name": "thread", "value": "26024-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_DNC"}]}